model/channel.go

```diff
@@ -206,6 +206,10 @@ func (o *Channel) IsGroupOrDirect() bool {
 	return o.Type == CHANNEL_DIRECT || o.Type == CHANNEL_GROUP
 }
 
+func (o *Channel) IsOpen() bool {
+	return o.Type == CHANNEL_OPEN
```

Comment:
:+1: -- might we add a unit test for coverage here?

Reply:
Of course! I thought I would be testing the exact implementation, not the behaviour itself, and just did as with `IsGroupOrDirect`, which is not tested.

Reply:
That's perfectly fair and a good counter argument. Fine to leave as-is :)